iceoryx_hoofs/functional/include/iox/detail/storable_function.inl

```diff
@@ -309,7 +317,9 @@ template <uint64_t Capacity, typename ReturnType, typename... Args>
 template <typename T>
 inline constexpr uint64_t storable_function<Capacity, signature<ReturnType, Args...>>::required_storage_size() noexcept
 {
-    return sizeof(T) + alignof(T) - 1;
+    const auto size{sizeof(T)};
+    const auto alignment{alignof(T)};
+    return size + alignment - 1;
```

Comment:
This helps to suppress the warning about missing parentheses? Oh boy...

Reply:
Well, it's the evaluation order which is now defined :sweat_smile:

Reply:
Actually it was just an unreliable scan :/